openshift/rosa
# SDA-4457 : Fix support for conditional addon parameters

Fixes : https://issues.redhat.com/browse/SDA-4457

This change : 
- Bumps ocm-sdk to v0.1.197
- Adds `resource` type to addon parameter interactive menu
- Replaces getAddonParameter call to use `addon-inquiries` endpoint

To verify 
- Create ROSA cluster
- Add Managed API Addon
```
rosa install addon managed-api-service --cluster=<<cluster-name>> --addon-managed-api-service=1 --addon-resource-required=true --cidr-range=10.1.0.0/26 -y 
```
- Verify no errors, and addon is created
